docs/source/create_channel/create_channel_participation.md

```diff
@@ -0,0 +1,550 @@
+```
+
+Replace:
+
+- `CHANNEL_NAME` with the name of the channel specified in the `configtxgen` command when you generated the channel.
+- `ORDERER_ADMIN_LISTENADDRESS` with Orderer.Admin.ListenAddress defined in the orderer.yaml for this orderer.
+- `OSN_TLS_CA_ROOT_CERT` with the path and file name of the orderer organization TLS CA root certificate and intermediate certificate if using an intermediate TLS CA.
+- `ADMIN_TLS_SIGN_CERT` with the path and file name of the admin client signed certificate from the TLS CA.
+- `ADMIN_TLS_PRIVATE_KEY` with the path and file name of the admin client private key from the TLS CA.
+
+The output of this command looks similar to:
+```
+{
+  "name": "channel1",
+  "url": "participation/v1/channels/channel1",
+  "consensusRelation": "consenter",
+  "status": "active",
+  "height": 1
+}
+```
+
+The following diagram summarizes the steps you have completed:
+
+![create-channel.osnadmin1-3](./osnadmin1-3.png)
+_You used the configtxgen command to create the channel genesis block and provided that file when you ran the osnadmin channel join command for each orderer by targeting the admin endpoint on each node._
+
+Assuming you have successfully run the `osnadmin channel join` on all three ordering nodes, you now have an active channel and the ordering service is ready to order blocks. Peers can join the channel and begin to transact.  
+
+If you want to join additional ordering nodes to the consenter set of the channel, follow the instructions in the next section.
+
+## Step three: Join additional ordering nodes
+
+Over time it may be necessary to add additional ordering nodes to the consenter set of a channel. Other organizations may want to contribute their own orderers to the cluster, or it might prove advantageous under production conditions to permit multiple orderers to go down for maintenance at the same time.
+
+Anytime a new orderer joins the cluster, you need to ensure that you do not lose quorum while the ledger on the new orderer catches up. Adding a fourth orderer to an existing three node cluster changes the majority from two to three. Therefore, during the catch-up process and until the new orderer starts executing consensus, the cluster is in a state of reduced fault tolerance. To account for this situation and avoid any downtime, a command that can ascertain the `consensusRelation` status of an orderer, which can be either "consenter" or "follower", has been introduced. When a node joins as a follower, the channel ledger is replicated from other orderers, but as it does not take part in consensus, channel operations are not impacted. Because replicating a long chain of blocks could take a long time, joining as a follower is useful for channels with high block heights. To join an orderer to a channel as a follower, **do not include the node in the channel configuration consenter set**.
```

Comment:
> Because replicating a long chain of blocks could take a long time, joining as a follower is useful for channels with high block heights.

I would say "is most useful".  But it's generically useful because it allows you to confirm that an OSN is able to replicate blocks before having it count towards quorum.  (You can imagine for instance, a network firewall rule which prevents the orderer from contacting the others being detectable in this way).

Reply:
Good point. I changed it to
Because replicating a long chain of blocks could take a long time, joining as a follower is most useful for channels with high block heights because it allows you to confirm that an orderer is able to replicate blocks before having it count towards quorum.